src/views/LoginFrame.java

```diff
@@ -133,21 +123,19 @@ private void LoginButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-F
         if (LabarotoryJournal.con.checkUserAuth(username, password)) {
             if (LabarotoryJournal.con.checkUserAdmin(username)) {
                 //TODO: Opens AdminFrame
+                isAdmin = true;
+                new MenuFrame().setVisible(true);
             }
             else {
-                new ShowJournal().setVisible(true);
+                isAdmin = false;
+                new MenuFrame().setVisible(true);
```

Comment:
Я планував, що параметр **IsAdmin** буде передаватись в **MenuFrame()**, тобто **MenuFrame(IsAdmin)**

```suggestion
                new MenuFrame(IsAdmin).setVisible(true);
```